### PR TITLE
feat: added support for azure dc in contentstack package

### PR DESCRIPTION
**Pull request by vkalta**

### CODE REVIEW

**Review by abhinav-from-contentstack — COMMENTED** [review]
please add AZURE as an argument in paranthesis, like ( EU | NA | AZURE )
in this file packages/contentstack/README.md

**Comment by netrajpatel** [comment]
@shafeeqd959 @vkalta, @abhinav-from-contentstack 
Just review this PR, and not merge this till we get the green from QA.

@vkalta You can add your further commits in same branch and ask QA to test it from your branch itself.

**Comment by shafeeqd959** [comment]
@netrajpatel , Reviewed this PR and added the comments and informed @vkalta. Thanks

**Review by shafeeqd959 — COMMENTED** [review]
update the versions of the package which has changes,
Update the region from AZURE to AZURE-NA

File: packages/contentstack/README.md [hunk 1]
```diff
@@ -1135,7 +1135,7 @@ USAGE
   $ csdx config:set:region [REGION]
 
 ARGUMENTS
-  REGION  (EU|NA) North America(NA), Europe (EU)
+  REGION  (EU|NA) North America(NA), Europe (EU), AZURE
```
[the remark this answers is not in the document]
Comment (vkalta) [reply packages/contentstack/README.md]: Changed everywhere. Thanks.

File: packages/contentstack-export/README.md [hunk 2]
```diff
@@ -10,6 +10,25 @@ It is Contentstack’s CLI plugin to export content from the stack. To learn how
 * [Commands](#commands)
 <!-- tocstop -->
 
+For switching to EU region update the hosts at config/default.js
+
```
[the remark this answers is not in the document]
Comment (vkalta) [reply packages/contentstack-export/README.md:14]: Done. Thanks for reviewing.

**Pull request merged**